FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -84,6 +84,16 @@ and both versions should be accepted.
 
 It should be noted that the `fetch()` request using `adAuctionHeaders` can also be used to send `auctionBlob` (e.g. in the request body) and receive the response blob (e.g. in the response body).
 
+#### Alternate header to facilitate delaying response blobs to response body
+
+The aforementioned Step 3 includes the hash of the response blob in an `Ad-Auction-Result` HTTP response header, which requires that the response blob be available before the HTTP response headers are sent back to the device.  If the server operator wants to instead delay inclusion of the response blob until the HTTP response body is sent back to the device, they can instead omit the `Ad-Auction-Result` HTTP header and instead:
```

Comment:
It would be useful to explicitly clarify _why_ the server operator might delay inclusion of the response blob, either inlined here or by linking to additional documentation for context. As written, the latency implications and data dependency in common ad server flows is unclear.